x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryFolderTests.java

```diff
@@ -195,8 +195,8 @@ public void testLocalExecWithCountAndWhereFalseFilter() {
         PhysicalPlan p = plan("SELECT COUNT(10), COUNT(DISTINCT 20) WHERE 1 = 2" + randomOrderByAndLimit(2));
         assertEquals(LocalExec.class, p.getClass());
         LocalExec le = (LocalExec) p;
-        assertEquals(EmptyExecutable.class, le.executable().getClass());
-        EmptyExecutable ee = (EmptyExecutable) le.executable();
+        assertEquals(SingletonExecutable.class, le.executable().getClass());
```

Comment:
Why did the empty executable was replaced by singleton? Was the initial test wrong?

Reply:
yes, that's one of the cases where SQL engines return one row with an "empty" result. E.g. Postgres:
```
lukas=# SELECT COUNT(10), COUNT(DISTINCT 20) WHERE 1 = 2;
 count | count
-------+-------
     0 |     0
(1 row)
```